feature/bgp/static_route_bgp_redistribution/otg_tests/static_route_bgp_redistribution_test/static_route_bgp_redistribution_test.go

```diff
@@ -479,6 +489,17 @@ func configureTableConnection(t *testing.T, dut *ondatra.DUTDevice, isV4, mPropa
 	}
 
 	batchSet.Set(t, dut)
+	if deviations.TcMetricPropagationUnsupported(dut) {
+		med := medZero
```

Comment:
This needs to use  switch dut.Vendor() format.  See the requirements for deviations at:

https://github.com/openconfig/featureprofiles/blob/f80fb73ceb4b397b4c1cf6a7b373d1421650d0e9/internal/deviations/README.md?plain=1#L182-L187

(This is because the CLI you are adding is specific to Cisco.  Other vendors may also use this deviation, but with their own CLI)